frontend/src/pages/Auth/Firebase.tsx

```diff
@@ -237,8 +236,7 @@ export const Firebase: React.FC = () => {
 						{params.mode === 'resetPassword' ? (
 							<ResetPassword formStore={formStore} />
 						) : null}
-						{params.mode === 'recoverEmail' ||
-						params.mode === 'verifyEmail' ? (
```

Comment:
This could be cause I tried to test locally and the confirmation link was a prod link but got stuck on this page on the confirmation link

<img width="442" alt="Screenshot 2024-07-01 at 12 38 10 PM" src="https://github.com/highlight/highlight/assets/17744174/ff0aaa40-8c72-4f77-af85-57f1798ea550">

1. Create new email with password
2. Confirm email
3. Link in email links to page above and is stuck, but the other previously opened page redirects to About you"

Reply:
when testing this, did you click the confirmation email link as is (to app.highlight.io), or did you update the URL to localhost?

Reply:
It was to `app.highlight.io`

Reply:
ah yes, if you were to load the reflame preview you'd see the behavior shown in the loom